lib/package_json.test.ts

```diff
@@ -42,6 +42,7 @@ Deno.test("single entrypoint", () => {
       version: "0.1.0",
     },
     testEnabled: true,
+    includeEsModule: true,
```

Comment:
We should add a test here that sets this to `false`.

Reply:
Done.